stdlib/sql/from.go

```diff
@@ -173,7 +173,7 @@ func (c *sqlIterator) Do(ctx context.Context, f func(flux.Table) error) error {
 
 	rows, err := db.QueryContext(ctx, c.spec.Query)
 	if err != nil {
-		return err
+		return errors.New(codes.Invalid, err)
```

Comment:
I think `errors.Wrap(err, codes.Invalid)` is more appropriate.

Reply:
Fixed